python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -149,6 +165,8 @@ class GrapheneAssetNode(graphene.ObjectType):
     graphName = graphene.String()
     groupName = graphene.String()
     id = graphene.NonNull(graphene.ID)
+    is_source = graphene.NonNull(graphene.Boolean)
```

Comment:
nit: use snakecase for resolvers (`is_source` -> `isSource`)

Reply:
good catch